core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/TinyLfuBlockCache.java

```diff
@@ -48,12 +52,13 @@
   private final Cache<String,Block> cache;
   private final Policy.Eviction<String,Block> policy;
   private final ScheduledExecutorService statsExecutor;
+  private final Map<String,CacheEntry> weakEvictionMap = new MapMaker().weakValues().makeMap();
```

Comment:
FYI, Caffeine also supports weak values so you could use two instances. It has better concurrency and creates less garbage, whereas Guava's has slightly better memory efficiency due to forking the hash table.